docs/samples/interaction/cartesianplane.md

```diff
@@ -0,0 +1,263 @@
+# Cartesian plane
+
+```js chart-editor
+// <block:setup:5>
+let RECT;
```

Comment:
Since it is not a final/const I would use lower case for this

Reply:
Thank you @LeeLenaleee ! I left it uppercase to highlight it ;) I do

Reply:
What do you think about position on the menu (in "interaction", maybe "miscellaneous" is better?) and about the sample itself? to complex? I had some doubts...